### PR TITLE
feat(sklearn): load models using cloudpickle to support module import

### PR DESCRIPTION
**Pull request by matbleu**
Load sklearn model using cloudpickle library, to support deserialization of pickled files containing sklearn + module pickled by value (cf https://github.com/cloudpipe/cloudpickle#overriding-pickles-serialization-mechanism-for-importable-constructs). 
Use-cases : a custom sklearn estimator needs to be served, but the estimator class being defined needs to be imported from a module before it is unpickled.

+ update sklearn version to 1.1.0

### CODE REVIEW

File: src/pythie_serving/sklearn_wrapper.py [hunk 1]
```diff
@@ -18,7 +18,8 @@ def __init__(self, *, logger: logging.Logger, model_server_config: model_server_
         self.model_map = {}
         for model_config in model_server_config.model_config_list.config:
             with open(os.path.join(model_config.base_path, model_config.name) + ".pickled", 'rb') as opened_model:
-                model = pickle.load(opened_model)
+                # cloudpickle used to be able to load model + modules which are not importable
+                model = cloudpickle.load(opened_model)
```
[the remark this answers is not in the document]
Comment (crqs) [reply src/pythie_serving/sklearn_wrapper.py:22]: sure! I already wrote an issue about this (CI/documentation/unittests)

File: src/pythie_serving/sklearn_wrapper.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 from .tensorflow_proto.tensorflow_serving.apis import predict_pb2, prediction_service_pb2_grpc
 from .utils import parse_sample
 from .exceptions import PythieServingException
-
+import cloudpickle
```
[the remark this answers is not in the document]
Comment (matbleu) [reply src/pythie_serving/sklearn_wrapper.py:12]: Indeed, in that case I can directly add isort + black to the CI + a pre-commit yaml. Or is there a particular reason they were not added ?

[the remark this answers is not in the document]
Comment (cp2587) [reply src/pythie_serving/sklearn_wrapper.py:12]: just a lack of time - you can add them

File: src/pythie_serving/sklearn_wrapper.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (matbleu) [reply src/pythie_serving/sklearn_wrapper.py:32]: Yes that's also my understanding + how i experienced it in local

[the remark this answers is not in the document]
Comment (crqs) [reply src/pythie_serving/sklearn_wrapper.py:32]: (aha I read your message too fast, didn't plan anything, I'll add an issue for this)

**Pull request merged**